ActivityHistoryApi.Tests/DynamoDbIntegrationTests.cs

```diff
@@ -1,58 +1,73 @@
 using Amazon.DynamoDBv2;
 using Amazon.DynamoDBv2.DataModel;
-using NUnit.Framework;
+using Amazon.DynamoDBv2.Model;
 using System;
 using System.Collections.Generic;
 using System.Net.Http;
+using Xunit;
 
 namespace ActivityHistoryApi.Tests
 {
-    public class DynamoDbIntegrationTests<TStartup> where TStartup : class
+    public class DynamoDbIntegrationTests<TStartup> : IDisposable where TStartup : class
     {
-        protected HttpClient Client { get; private set; }
-        private DynamoDbMockWebApplicationFactory<TStartup> _factory;
-        protected IDynamoDBContext DynamoDbContext => _factory?.DynamoDbContext;
-        protected List<Action> CleanupActions { get; set; }
+        public HttpClient Client { get; private set; }
+        public IDynamoDBContext DynamoDbContext => _factory?.DynamoDbContext;
 
+        private readonly DynamoDbMockWebApplicationFactory<TStartup> _factory;
         private readonly List<TableDef> _tables = new List<TableDef>
         {
-            // TODO: Populate the list of table(s) and their key property details here, for example:
-            //new TableDef { Name = "example_table", KeyName = "id", KeyType = ScalarAttributeType.N }
+            new TableDef
+            {
+                Name = "ActivityHistory",
+                KeyName = "targetId",
+                KeyType = ScalarAttributeType.S,
+                RangeKeyName = "id",
+                RangeKeyType = ScalarAttributeType.S,
+                LocalSecondaryIndexes = new List<LocalSecondaryIndex>(new[]
+                {
+                    new LocalSecondaryIndex
+                    {
+                        IndexName = "ActivityHistory",
```

Comment:
This is not a great index name. It should be something like: `ActivityHistoryByCreatedAt`
This should of course match the index definition you have in the terraform files.